fifa_ratings_predictor/data_methods.py

```diff
@@ -1,78 +1,86 @@
+import datetime
 import glob
 import json
-import datetime
 
-import pandas as pd
+import fifa_ratings_predictor.constants as constants
```

Comment:
Should this one not be at the bottom?

Reply:
https://github.com/timothycrosley/isort doesn't move it?  🤷‍♀️

Reply:
Hmmmmm. Ok.